action.yml

```diff
@@ -5,21 +5,21 @@ inputs:
     description: 'Which directory to upload'
     required: true
     default: 'exported-artifacts'
-  distro:
-    description: 'The distribution of artifacts'
-    required: true
-    default: 'el8stream'
 runs:
   using: "composite"
   steps:
     - name: Install createrepo_c if needed
       shell: bash
       run: which createrepo_c || dnf install -y createrepo_c
-    - name: Create DNF repository
+    - name: Create DNF repository if needed
+      shell: bash
+      run: test -d ${{ inputs.directory }}/repodata || createrepo_c ${{ inputs.directory }}
```

Comment:
Probably worth to mention that feature in the PR message too

Reply:
Done